Lib/cgi.py

```diff
@@ -351,10 +352,14 @@ def __init__(self, fp=None, headers=None, outerboundary=b'',
             for the page sending the form (content-type : meta http-equiv or
             header)
 
+        max_num_fields: Integer. If set, then __init__ throws a ValueError
```

Comment:
If you really want a type annotation (of which I see no precedent above), then just write "int".

Reply:
Thanks, fixed in ab0eb93f21.